src/fake-timers-src.js

```diff
@@ -309,14 +310,18 @@ function withGlobal(_global) {
     }
 
     /**
-     * @param clock
-     * @param job
+     * @param {Clock} clock
+     * @param {Timer} job
      */
     function getInfiniteLoopError(clock, job) {
         const infiniteLoopError = new Error(
             `Aborting after running ${clock.loopLimit} timers, assuming an infinite loop!`
         );
 
+        if (!job.error) {
+            return infiniteLoopError
```

Comment:
semicolon?

Reply:
hah, no CI means reviewer needs to notice lint errors 🙈

Reply:
wait, there's no lint rule for semis 🤦